### DISCUSSION TITLE
How to define location of generated tftui.plan

### DISCUSSION THREAD

**Discussion by remiflament** [idoavrah/terraform-tui#83] [Q&A] [answered]
Hello 👋🏻 Thanks for this usefull tool !
Do you have already implemented a configuration file that can be use ? example `~/user/.config/tftui.config`

I was looking on a way to store the `tftui.plan `under the folder `.terraform/` despite of the root folder actually use.

Thanks you

**Comment by idoavrah** [comment, marked as answer]
I actually forgot this was a request :)

I'll create an issue and try to find the time to implement it. Sounds useful.
